package.json

```diff
@@ -15,19 +15,17 @@
     "firebase": "8.3.3",
     "i18next-browser-languagedetector": "6.1.4",
     "idb-keyval": "6.0.3",
-    "jotai": "1.13.1",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
```

Comment:
@ad1992 did we define these packages in root for a specific reason? I don't recall.

But if we move these to the excalidraw-app, we should do it alongside the other deps that only the excalidraw-app is using, such as `idb-keyval`, `firebase`, etc. Pretty much all the packages listed here in fact.

Reply:
I made the changes in my latest commit

Reply:
Ideally it should be in the app folder instead of root but I was facing some issues with build due to which I couldn't move it. However there were lots of fixes pushed related to workspaces post that so probably this issue might have gotten fixed and if all is working fine we should be able to move it to app. I will test it before merging